Bosh exporter all envs

m-richo opened the pull request:
Sorry it's a largish PR.  let me know your thoughts.
concourse Pipeline has been reverted to deploy test env off master branch.

Commit:
add sandpit testing to test deployment.
run fly format-pipeline on pipeline.yml

Commit:
use bosh-dns on prometheus deployment

Commit:
update pipeline to use new operators.  fly format pipeline.yml. add some comments to operator files

Commit:
Update documentation and rename operator files to be more intuitive

Commit:
keep consistent naming between pipelines.  use d-cld, y-cld and b-cld instead of sandpit, staging and prod

gordcorp commented on README.md:
```diff
@@ -5,23 +5,64 @@ Prometheus is an Open source monitoring tool.  A BOSH release was created for it
 
 
-## Usage
+# Usage
```
I'm going to be THAT guy - there's actually a WCAG accessibility thing that a page should only contain one h1

Commit:
have only 1 H1 heading in readme.
update pipeline to use correct operator filenames.
correct label for d and b clouds.
